docs/configuration.md

```diff
@@ -114,6 +114,7 @@ information about each option that appears later in this page.
           age: 168h
           interval: 24h
           dryrun: false
+        readonly: false
```

Comment:
I'd rather not have the environment variable work that have to migrate the configuration format. We can always fix the env bug.